modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/internal/importer/MediaWikiImporter.java

```diff
@@ -66,20 +65,24 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import org.osgi.service.component.annotations.Activate;
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Reference;
-
 /**
  * @author Alvaro del Castillo
  * @author Jorge Ferrer
  */
-@Component(
-	configurationPid = "com.liferay.wiki.configuration.WikiGroupServiceConfiguration",
-	service = MediaWikiImporter.class
-)
 public class MediaWikiImporter {
```

Comment:
Just specify why you used Util classes instead of using references via parameter